### ISSUE TITLE
Conditions sorting not 1 to 10

### ISSUE THREAD

**Issue by Zidras** [Bunny67/WeakAuras-WotLK#46]
## Describe the bug

This is completely nitpicking, just wanted to report the issue. Conditions are sorting by first digit.  Using beta, could you check if this is also in Shadowlands?

### To Reproduce

1. Create 10+ Display elements (in my case texts)
2. Open Conditions tabs
3. Check the "Then"

**Screenshots**

![image](https://user-images.githubusercontent.com/10605951/91080909-4ba9c700-e63e-11ea-9269-69f99969dbd8.png)

**Which version of WeakAuras are you using?**

Beta

**Comment by Bunny67** [comment]
![image](https://user-images.githubusercontent.com/5816685/91123628-03be8a80-e6a6-11ea-9760-629028dfbdc7.png)

**Comment by Zidras** [comment]
Yeah you have the same as me. Would make more sense if it was 1,2,3,4,5,6,7,8,9,10,11...
